baconTests/StorageManagerTests.swift

```diff
@@ -15,18 +15,89 @@ class StorageManagerTests: XCTestCase {
         XCTAssertNoThrow(try StorageManager())
     }
 
+    func test_getNumberOfTransactionsInDatabase() {
+        // swiftlint:disable force_try
+
+        let database = try! StorageManager()
+        XCTAssertNoThrow(try database.clearTransactionDatabase())
+        // Empty database should return 0 for getNumberOfTransactionsInDatabase()
+        XCTAssertEqual(database.getNumberOfTransactionsInDatabase(), 0)
+        // Add some transactions into database
+        XCTAssertNoThrow(try database.saveTransaction(TestUtils.validTransactionExpenditure01))
+        XCTAssertNoThrow(try database.saveTransaction(TestUtils.validTransactionExpenditure02))
+        XCTAssertEqual(database.getNumberOfTransactionsInDatabase(), 2)
+
```

Comment:
after we have methods like deleting/editing, can expand here by testing whether the number is still correct after deleting/editing

Reply:
noted!